src/main/resources/model/bibframe_authority.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "bibframe-authority",
+  "eventBodyJavaClass": "org.folio.search.domain.dto.BibframeAuthority",
+  "languageSourcePaths": ["$.languages"],
```

Comment:
You specify that your schema has `languages` field that could be used then for multilang index. But I can't see the field in bibframeAuthority schema.
If there is no such field in schema then it doesn't have sense to use multilang index.